dsnp-wallet/dsnp-wallet/Source/Extensions/UIImage+Cropped.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
For now let's remove this class until we know how we're going to handle images. For now let's just display the image that they've taken/selected in the imageView as contentMode aspectFill.